common/common.cpp

```diff
@@ -599,9 +603,9 @@ bool gpt_params_parse_ex(int argc, char ** argv, gpt_params & params) {
                 break;
             }
             params.main_gpu = std::stoi(argv[i]);
-#ifndef GGML_USE_CUBLAS
-            fprintf(stderr, "warning: llama.cpp was compiled without cuBLAS. Setting the main GPU has no effect.\n");
-#endif // GGML_USE_CUBLAS
+#ifndef GGML_USE_CLBLAS_SYCL
+            fprintf(stderr, "warning: llama.cpp was compiled without cuBLAS/SYCL. Setting the main GPU has no effect.\n");
+#endif // GGML_USE_CLBLAS_SYCL
```

Comment:
The name of the macro is `GGML_USE_CUBLAS_SYCL` (not `CLBLAS`). There a few other cases in this file.

Reply:
yes. it's bug when rebase the code.

Reply:
fix in https://github.com/ggerganov/llama.cpp/pull/2690/commits/fb15de38efdecab93d73cb52d413d32ea2f38924